<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionForm</class>
 <widget class="QWidget" name="SelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>2</number>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>278</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="sortButton">
      <property name="text">
       <string>Sort</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sortAscending">
      <property name="text">
       <string>Ascending</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sortDescending">
      <property name="text">
       <string>Descending</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>440</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addFilesButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Add individual files</string>
      </property>
      <property name="text">
       <string>Add Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>or</string>
      </property>
      <property name="margin">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addDirButton">
      <property name="toolTip">
       <string>Add directory contents</string>
      </property>
      <property name="text">
       <string>Add Directory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filterIncludeFiles">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Files</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filterIncludeDirs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Folders</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="addRecursive">
      <property name="text">
       <string>Recursive</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="filterNamesLabel">
      <property name="text">
       <string>Filters:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filterNames">
      <property name="toolTip">
       <string>Space separated list of filters</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filterIncludeHidden">
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>Hidden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
